### PR TITLE
fix(core): `Textfield` has change detection problems for `[filler]`

### LINKED ISSUE #9238: 🐞 - `Textfield` has change detection problems for filler
### Code Example URL

https://github.com/taiga-family/maskito/blob/time-meridiem-feature/projects/demo/src/pages/kit/time/examples/4-time-segments-min-max/component.ts

### Description

https://github.com/user-attachments/assets/b69561ad-0f0a-4390-8a18-92590a46aae4

### Reproduction
```html
<tui-textfield filler="HH:MM">
    <input
        tuiTextfield
        [(ngModel)]="value"
    />
</tui-textfield>
```

https://github.com/taiga-family/maskito/blob/755375e7f111e153c9e67f8ff12f35667efebe9c/projects/demo/src/pages/kit/time/examples/4-time-segments-min-max/component.ts#L14-L32

### Taiga UI version

4.8.1

### Browser Used

Chrome

### OS Used

macOS

### PR DESCRIPTION
**Pull request by nsbarsukov**
Fixes #9238

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
Could you add cypress component test?

**Comment by nsbarsukov** [comment]
> Could you add cypress component test?

I am not sure that this case could be covered by simple test with some assertion.
It should be at least screenshot test. Screenshot testing is not supported by our infrastructure.

File: projects/core/components/textfield/textfield.directive.ts [hunk 1]
```diff
@@ -35,6 +37,11 @@ export class TuiTextfieldBase<T> implements OnChanges {
     @Input()
     public invalid: boolean | null = null;
 
+    public nativeValue = toSignal(
+        fromEvent(this.el, 'input', () => this.el.value),
```
Comment (waterplea) [inline projects/core/components/textfield/textfield.directive.ts:41]: Didn't know you could add mapping function like that 😱

File: projects/core/components/textfield/textfield.component.ts [hunk 2]
```diff
@@ -81,6 +84,21 @@ export class TuiTextfieldComponent<T> implements TuiDataListHost<T> {
...
+        const value = this.directive?.nativeValue() || '';
+        const filledValue = value + this.filler().slice(value.length);
+
+        return filledValue.length > value.length ? filledValue : '';
+    });
+
+    protected showFiller = computed(() =>
+        Boolean(
+            this.focused() &&
+                this.computedFiller() &&
```
Comment (waterplea) [inline projects/core/components/textfield/textfield.component.ts:97]: Weird tabbing 🤔

**Review by waterplea — APPROVED** [review]

File: projects/core/components/textfield/textfield.component.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/core/components/textfield/textfield.component.ts:97]: Unfortunately, prettier does it (

https://github.com/user-attachments/assets/a06f4c4c-a307-405d-a285-c05886116f6d

The only way to bypass it is drop `Boolean` and add some `!!`

```ts
protected showFiller = computed(
    () =>
        this.focused() &&
        !!this.computedFiller() &&
        (!!this.directive?.nativeValue() || !this.input?.nativeElement.placeholder),
);
```

What do you think ?

Comment (waterplea) [reply projects/core/components/textfield/textfield.component.ts:97]: I guess it looks better

Comment (splincode) [reply projects/core/components/textfield/textfield.component.ts:97]: You can report this issue to [prettier](https://github.com/prettier/prettier)

Comment (nsbarsukov) [reply projects/core/components/textfield/textfield.component.ts]: > You can report this issue to [prettier](https://github.com/prettier/prettier)

@splincode done
https://github.com/prettier/prettier/issues/16709

**Pull request merged**